queries/wit/indents.scm

```diff
@@ -0,0 +1,16 @@
+[
+  (use_items)
+  (fields)
+  (variant_items)
+  (variant_payload)
+  (flags)
+  (enum_items)
+  (union_items)
+  (args)
+  (resource_items)
+] @indent
```

Comment:
So are these :)

Reply:
Thanks! Deleted for now as I'll focus on the indents first.